.github/ISSUE_TEMPLATE/issue------feature-request.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Quick clarification. Why do we need this file `issue------feature-request.md` and this file `feature_request.md`?

Reply:
yeah quite strange, these are auto generated files by github. probably something to do with how these templates work.

Reply:
yeah weird, I also don't see this files in pytorch's repo for example https://github.com/pytorch/pytorch/tree/master/.github/ISSUE_TEMPLATE